north/src/manifest.rs

```diff
@@ -12,7 +12,7 @@
 //   See the License for the specific language governing permissions and
 //   limitations under the License.
 
-use anyhow::{anyhow, Context, Error, Result};
+use anyhow::{Context, Result};
```

Comment:
Is this still used? If the api module uses `thiserror` then there shouldn't be a need for `anyhow`